lumigator/backend/backend/tests/conftest.py

```diff
@@ -59,6 +62,12 @@
 POLL_WAIT_TIME = 10
 
 
+def load_json_from_file(file_path: Path) -> dict:
+    """Load JSON data from a file path and return it as a dictionary."""
+    with Path.open(file_path) as file:
+        return json.load(file)
```

Comment:
Nitpick but this seems simple enough that it doesn't need to be it's own function, idk.

Reply:
I think it was a hangover from when I had lots more tests for the now reverted code.

To save me having to have these two lines in each test, I just added the fixture to the params for it. 🤔